store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java

```diff
@@ -2793,6 +2797,7 @@ public void notifyMessageArriveIfNecessary(DispatchRequest dispatchRequest) {
     class ReputMessageService extends ServiceThread {
 
         protected volatile long reputFromOffset = 0;
+        protected volatile long currentReputTimestamp = 0;
```

Comment:
Should the initial value be set to the current time? If it's set to 0, would it retrieve an unexpectedly large value before being assigned?

Reply:
it is ok